### PR TITLE
make target filepath directory dynamic rather than hardcoded

### PR DESCRIPTION
**Pull request by itscalvinwang**
[Ticket](https://www.notion.so/Strings-cli-translate-output-path-should-be-relative-to-the-input-path-21a35a50a0ff4730aaf16ff32439675c)

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -83,20 +83,25 @@ async fn main() {
...
-            let target_file_path = format!("strings/strings_{:#?}.json", target_language);
+            let parent_dir = source_filepath.parent().unwrap();
+
+            let target_filename = format!("strings_{:#?}.json", target_language);
+            let target_filepath = parent_dir.join("/").join(target_filename);
 
             let api_key = &args.api_key;
 
-            let source_json = match file::from_json(source_path.to_string()) {
+            let source_json = match file::from_json(source_filepath_str.to_string()) {
```
Comment (itscalvinwang) [inline src/main.rs:104]: Does `source_filepath_str` need to be used here or should it work with a Rust Path as well?

Comment (Snarik) [reply src/main.rs:104]: It's a good question. My gut wants to use a Path object here but I can't find anything that says this function will be okay with that. Let's stick to a string for now can clean up later.

**Review by Snarik — APPROVED** [review]
Lgtm

File: src/main.rs (hunk 1 above)
Comment (itscalvinwang) [reply src/main.rs:104]: I just realized that this is a function defined in our module and not a library. Holding off on refactoring that right now.

Comment (Snarik) [reply src/main.rs:104]: That would explain why I couldn't google it

**Comment by Snarik** [comment]
lmk when you've finished testing this so I can show you how to cut a release.

**Pull request merged**